topi/python/topi/transform.py

```diff
@@ -29,6 +30,48 @@ def _compute(*indices):
     return tvm.compute(new_shape, _compute)
 
 
+@tvm.tag_scope(tag=tag.BROADCAST)
+def expand_like(a, shape_like, axis, exclude):
+    """Expand the shape of an array.
+
```

Comment:
Give an example, as expand_like is not a numpy function

Reply:
Added.